src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -215,19 +211,24 @@ const styles = StyleSheet.create({
   promptTitle: {
     fontSize: 14,
     fontWeight: '600',
-    color: COLORS.white
+    color: COLORS.white2
   },
   container: {
-    backgroundColor: COLORS.default_signed_secondary,
+    backgroundColor: COLORS.signed_secondary,
     borderRadius: 16,
     paddingHorizontal: 12,
     paddingVertical: 12,
     marginTop: 10,
     paddingBottom: 50
   },
-  editPromptLabel: {color: COLORS.signed_secondary, textDecorationLine: 'underline'},
+  editPromptLabel: {
+    fontSize: 14,
+    fontFamily: fonts.inter[500],
+    color: COLORS.white2,
+    marginLeft: 4
+  },
   bioTextNull: {
-    color: COLORS.lightgrey,
+    color: COLORS.white2,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here involve updating the color values and font family used in the styles. These changes seem fine as long as they align with the new design requirements. However, I noticed that the `fontFamily` property for `bioTextNull` style is still set to 'Inter'. If you're using the `fonts` utility, it should be updated to use `fonts.inter[500]` or whatever weight is appropriate. 
```diff
- fontFamily: 'Inter',
+ fontFamily: fonts.inter[500],
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->